scalableminds/webknossos
# Update airbrake

### URL of deployed dev instance (used for testing):
- https://updateairbrake.webknossos.xyz

### Steps to test:
- trigger an exception (e.g., type in typeahead in task view) and check whether airbrake reported the error correctly

------
